### PR TITLE
Port v4 browser support table format to v3

### PR DESCRIPTION
**Pull request by patrickhlauke**
Port of https://github.com/twbs/bootstrap/pull/18614 from v4 to v3

Ref https://github.com/twbs/bootstrap/issues/19269

### CODE REVIEW

**Comment by patrickhlauke** [comment]
@mdo @cvrebert thoughts on this? I note that the v4 tables included Android browser/ webview, while the v3 table didn't. That's my only concern here (the rest should be fairly non-controversial)

File: docs/_includes/getting-started/browser-device-support.html [hunk 1]
```diff
@@ -21,21 +27,53 @@ <h2 id="support-browsers">Supported browsers</h2>
           <th scope="row">Android</th>
           <td class="text-success"><span class="glyphicon glyphicon-ok" aria-hidden="true"></span> Supported</td>
           <td class="text-success"><span class="glyphicon glyphicon-ok" aria-hidden="true"></span> Supported</td>
-          <td class="text-muted" rowspan="3" style="vertical-align: middle;">N/A</td>
-          <td class="text-danger"><span class="glyphicon glyphicon-remove" aria-hidden="true"></span> Not Supported</td>
           <td class="text-muted">N/A</td>
+          <td class="text-success"><span class="glyphicon glyphicon-ok" aria-hidden="true"></span> Supported</td>
```
Comment (cvrebert) [inline docs/_includes/getting-started/browser-device-support.html]: Should read "Android v5.0+ supported" to match v4

**Comment by cvrebert** [comment]
If we're going to claim support for Edge, then we should add Edge to v3's Sauce testing config.

I slightly question adding the "feature" of AndroidBrowser+Edge+Win10Mobile official support this late in v3's lifecycle though.

**Comment by patrickhlauke** [comment]
Fair enough about not adding new browser combos. As an alternative, then, how about just porting the top/tail text, split into mobile/desktop, but leave actual table content unchanged?

**Comment by mdo** [comment]
> Fair enough about not adding new browser combos. As an alternative, then, how about just porting the top/tail text, split into mobile/desktop, but leave actual table content unchanged?

Updated table layout and more granular text is a-okay by me. I'd rather not sign us up for more/new browser combos as @cvrebert mentioned :).

**Comment by cvrebert** [comment]
> As an alternative, then, how about just porting the top/tail text, split into mobile/desktop, but leave actual table content unchanged?

:+1:

**Comment by patrickhlauke** [comment]
rejigged the PR - this now ports the overall formatting, splitting the support table into separate mobile/desktop, but does not add any new browser combos (although I did add Microsoft Edge under the "Unofficially..." section, as from my experience it seems to all work fairly well there).

For a quick look of the end result of this, see http://www.bootply.com/oFgjmGsVGX

**Comment by twbs-lmvtfy** [comment]
Hi @patrickhlauke!

You appear to have posted a live example (http://s.bootply.com/render/oFgjmGsVGX), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 180, column 1 thru column 53: Attribute ```` http-equiv ```` not allowed on element ```` meta ```` at this point.
* line 180, column 1 thru column 53: Element ```` meta ```` is missing one or more of the following attributes: ```` itemprop ````, ```` property ````.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Pull request merged**